scripts/ci/sizes.py

```diff
@@ -109,21 +109,25 @@ def compare(previous_path: str, current_path: str, threshold: float) -> None:
             entries[name] = {}
         entries[name]["previous"] = entry
 
-    headers = ["Name", "Previous", "Current", "Change"]
+    headers = ["Name", before_header, after_header, "Change"]
     rows: list[tuple[str, str, str, str]] = []
     for name, entry in entries.items():
         if "previous" in entry and "current" in entry:
             previous = float(entry["previous"]["value"]) * DIVISORS[entry["previous"]["unit"]]
             current = float(entry["current"]["value"]) * DIVISORS[entry["current"]["unit"]]
 
-            min_change = previous * (threshold / 100)
-
             unit = get_unit(min(previous, current))
             div = get_divisor(unit)
 
-            change = ((previous / current) * 100) - 100
-            sign = "+" if change > 0 else ""
+            change = abs(((current - previous) / previous) * 100)
```

Comment:
Not sure if there's an easier way to preserve the `+` sign

Reply:
Use the `+` formatter when formatting:

```diff
- f"{sign}{change:.2f}%",
+ f"{change:+.2f}%",
```

Reply:
Also, you should name your variables more explicitly, e.g. `change_percent`